libs/UIHelpers/transfer_function_widget.cpp

```diff
@@ -135,135 +135,156 @@ void TransferFunctionWidget::draw_ui()
             }
           }
           selected_point = std::distance(alpha_control_pts.begin(), fnd);
         }
       }
       update_colormap();
     }
-    else if (ImGui::IsMouseClicked(1)) {
+    else if (ImGui::IsMouseClicked(1))
+    {
       selected_point = -1;
       // Find and remove the point
       auto fnd = std::find_if(
-        alpha_control_pts.begin(), alpha_control_pts.end(), [&](const vec2f& p) {
+          alpha_control_pts.begin(), alpha_control_pts.end(), [&](const vec2f &p)
+          {
           const vec2f pt_pos = p * view_scale + view_offset;
           float dist = (pt_pos - vec2f(io.MousePos)).length();
-          return dist <= point_radius;
-        });
+          return dist <= point_radius; });
       // We also want to prevent erasing the first and last points
       if (fnd != alpha_control_pts.end() && fnd != alpha_control_pts.begin() &&
-        fnd != alpha_control_pts.end() - 1) {
+          fnd != alpha_control_pts.end() - 1)
+      {
         alpha_control_pts.erase(fnd);
       }
       update_colormap();
     }
-    else {
+    else
+    {
       selected_point = -1;
     }
   }
 
   // Draw the alpha control points, and build the points for the polyline
   // which connects them
   std::vector<ImVec2> polyline_pts;
-  for (const auto& pt : alpha_control_pts) {
+  for (const auto &pt : alpha_control_pts)
+  {
     const vec2f pt_pos = pt * view_scale + view_offset;
     polyline_pts.push_back(pt_pos);
     draw_list->AddCircleFilled(pt_pos, point_radius, 0xFFFFFFFF);
   }
 
-
   draw_list->AddPolyline(polyline_pts.data(), polyline_pts.size(), 0xFFFFFFFF, false, 2.f);
   draw_list->PopClipRect();
 
-  //Add Label tick marks
+  // Add Label tick marks
   int nbTicks = 5;
   vec2f tick_pos = ImGui::GetCursorScreenPos();
   tick_pos.y -= ImGui::GetStyle().ItemSpacing.y;
   vec2f tick_size = ImGui::GetContentRegionAvail();
   tick_size.y = 5;
   draw_list->PushClipRect(tick_pos, ImVec2(tick_pos.x + tick_size.x, tick_pos.y + tick_size.y));
 
-  for (int i = 0; i < nbTicks; i++) {
+  for (int i = 0; i < nbTicks; i++)
+  {
     float percentage = float(i) / (nbTicks - 1);
 
     draw_list->AddLine(ImVec2(tick_pos.x + percentage * (tick_size.x - 1), tick_size.y),
-      ImVec2(tick_pos.x + percentage * (tick_size.x - 1), tick_pos.y + tick_size.y), ImColor(255, 255, 255, 255), 1);
+                       ImVec2(tick_pos.x + percentage * (tick_size.x - 1), tick_pos.y + tick_size.y), ImColor(255, 255, 255, 255), 1);
   }
   draw_list->PopClipRect();
 
-  //Add Label text
+  // Add Label text
   const float ItemSpacing = ImGui::GetStyle().ItemSpacing.x;
-  for (int i = 0; i < nbTicks; i++) {
+  for (int i = 0; i < nbTicks; i++)
+  {
     float percentage = float(i) / (nbTicks - 1);
     float val = (m_min_max_val[1] - m_min_max_val[0]) * percentage + m_min_max_val[0];
     std::stringstream text;
     text << std::fixed << std::setprecision(4) << val;
-    if (i == 0) {
+    if (i == 0)
+    {
     }
```

Comment:
Should there be logic here? An exception or a comment to say do nothing?

Reply:
This is code from the [transfer function widget repo](https://github.com/Twinklebear/imgui-transfer-function). I created an issue to update that library https://github.com/brown-ccv/VR-Volumeviewer/issues/49